.github/workflows/cd.yml

```diff
@@ -47,15 +56,25 @@ jobs:
         run: tb diff --no-verbose
       - name: Push changes to Workspace
         run: |
-            PUSH_FILE=./deployment/${{env.GIT_BRANCH}}-push.sh
-            if [ -f "$PUSH_FILE" ]; then
-              ./deployment/${{env.GIT_BRANCH}}-push.sh
+            source .tinyenv
+            CI_DEPLOY_FILE=./deploy/${VERSION}/ci-deploy.sh
+            if [ -f "$CI_DEPLOY_FILE" ]; then
+              ./deploy/${VERSION}/ci-deploy.sh
             else
-              if ${{ inputs.tb_deploy}}; then
-                tb deploy --populate --wait
-              else
-                tb push --push-deps --only-changes --force --populate --wait
-              fi
+              tb deploy --populate --wait
```

Comment:
Why not keeping the option not to use `tb_deploy` if not passing the param?

Reply:
Changed it!

I've removed it from the CI template (both github and gitlab), since we can just use tb deploy there (or the custom shell script).

For the case of CD, if not `tb_deploy` it runs `tb deploy` unless `deploy/VERSION/cd-deploy.sh` exists.

In Gitlab I'm using a TB_DEPLOY env variable with the same logic.